src/user/introduction.rst

```diff
@@ -89,7 +89,7 @@ From now on using ``conda install <package-name>`` will also find packages in ou
   For certain platforms (such as ARMv8 64-bit, formally known as `aarch64`)
   Anaconda, Inc. does not provide any Miniconda installers, however Miniforge
   does.
-  Miniforge installers are available here: https://github.com/conda-forge/miniforge/#download
+  Miniforge installers are available `here` <https://github.com/conda-forge/miniforge/#download>_.
```

Comment:
Oh, I missed this one. That second backtick (`) should be between the > and the _ to make the link correct.

Reply:
Done, thanks for pointing this out! Sorry for such a silly mistake, when I made these changes I didn't know how to check these locally before pushing. Won't repeat it! :)

Reply:
It happens to all of us! Not a problem.